cars/v1/vanilla/config.ini

```diff
@@ -16,7 +16,7 @@ jdk.unbundled.release_url = https://artifacts.elastic.co/downloads/elasticsearch
 
 docker_image=docker.elastic.co/elasticsearch/elasticsearch
 # major version of the JDK that is used to build Elasticsearch
-build.jdk = 17
+build.jdk = 18
```

Comment:
I wouldn't change this (yet) until it gets updated in the upstream repo in https://raw.githubusercontent.com/elastic/elasticsearch/master/.ci/java-versions.properties

The release/snapshot artifacts are compiled using the build version mentioned there and while performance regressions almost exclusively would be due to a _runtime_ jdk change, it's a good practice that Rally builds artifacts using the same Java version.

Reply:
Thanks, I reverted that part